.github/workflows/gh-pages.yml

```diff
@@ -1,10 +1,8 @@
-name: github pages
+name: Deploy public documentation
 
 on:
   push:
-    branches:
-      - main  # Set a branch to deploy
-  pull_request:
+    tags: '*'
```

Comment:
question(non-blocking): do we want to prefix releases so that this will not trigger anytime a tag is pushed in the repository?

Reply:
It's the same as we do for [production release](https://github.com/aws-samples/pcluster-manager/blob/main/.github/workflows/production_release.yml): since all tags are protected (meaning they can be created or removed only by maintainers) I don't think it's a real issue. GitHub creates automatically a tag when opening a new release, so no one has to create or update a tag by hand.